src/common/extension-telemetry-events.ts

```diff
@@ -283,6 +284,13 @@ export type AutoDetectedFailuresDialogStateTelemetryData = {
     enabled: boolean;
 };
 
+export type UnhandledExceptionTelemetryData = {
+    message: string;
+    stackTrace?: string;
+    source: string;
+    errorType: string;
```

Comment:
This can probably be a string enum, rather than a generic string (eg `'logger.error' | 'console.error' | 'unhandled error' | 'unhandled promise rejection'`)